knative-operator/deploy/crds/operator_v1alpha1_knativekafka_crd.yaml

```diff
@@ -76,7 +76,7 @@ spec:
                       transitioned from one status to another. We use VolatileTime
                       in place of metav1.Time to exclude this from creating equality.Semantic
                       differences (all other things held constant).
-                    type: object
+                    type: string
```

Comment:
This was failing with type object, although operator-sdk generated it with type object. I don't know :shrug:

Reply:
How was it failing?

Reply:
Was getting this:

```
Invalid value: \"string\": status.conditions.lastTransitionTime in body must be of type object: \"string\""
```

I have seen this before, the fix was changing to string.

Reply:
ack, it's in line with our other CRDs too.